circle.yml

```diff
@@ -18,6 +18,10 @@ test:
     - pip install coala-bears colorama==0.3.6 --pre -U:
         parallel: true
         timeout: 900  # Allow 15 mins
+    # Temporary workaround for
```

Comment:
So even this didnt work.
Need to wait until https://github.com/coala/coala-bears/issues/1110 is resolved.